downloadDirectory/src/test/resources/db.properties

```diff
@@ -0,0 +1,5 @@
+db.user=root
+db.password=root
+db.host=localhost
+db.port=3306
+db.name=release_current_before_addlinks_69
```

Comment:
Should this db.name property have a hard-coded release number?

Reply:
That's a file used for testing. I would expect that other developers would update as necessary.